Portfolio.html

```diff
@@ -69,39 +71,27 @@ <h4>January 2021, Bristol UK</h4>
 
 
 	<div class="aboutme" id="aboutme">
```

Comment:
The background picture in AboutMe is repeated when the browser is wider - not sure if this is what you wanted?
<img width="1329" alt="Screenshot 2020-09-08 at 14 13 56" src="https://user-images.githubusercontent.com/65677163/92480974-8df60b00-f1dd-11ea-8b86-76e3809e3a89.png">

Reply:
Bugger! I'd missed the hypen in "no repeat". Good catch

Reply:
Look good. However noticed in Chrome & Safari that zooming into 90 or 100% causes picture of you to move up ontop of the portfolio pics and the words get moved around too
<img width="1387" alt="Screenshot 2020-09-08 at 15 21 17" src="https://user-images.githubusercontent.com/65677163/92488836-f695b580-f1e6-11ea-846f-589a03a14173.png">

Reply:
I believe it was a rogue media query which now should be fixed

Reply:
Looking good now, yaay!

Reply:
Actually Safari still does not like it, your pic overlaps the portfolio boxes :(
<img width="1190" alt="Screenshot 2020-09-08 at 16 06 17" src="https://user-images.githubusercontent.com/65677163/92494187-4ecfb600-f1ed-11ea-8e57-d91bbae4ed66.png">